package.json

```diff
@@ -22,11 +22,13 @@
     "build": "npm run compile",
     "clean": "rm -rf ./build/",
     "compile": "tsc -p . --rootDir . --outDir build/",
+    "format-check": "echo \"TODO\"",
```

Comment:
This can be dangerous if the user has unsaved edits in their editor. A 'test' job should not make edits IMO.